src/common/components/Alert/Alert.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
```

Comment:
This component is super well done! I really like your use of `classNames`, props, default props, and accessibility. Fantastic work!